modules/images-configuration-registry-mirror.adoc

```diff
@@ -0,0 +1,182 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration.adoc
+
+[id="images-configuration-registry-mirror_{context}"]
+= Configuring image registry repository mirroring
+Setting up container registry repository mirroring lets you:
+
+* Configure your {product-title} cluster to redirect requests
+to pull images from a repository on a source image registry and have
+it resolved by a repository on a mirrored image registry.
+* Identify multiple mirrored repositories for each target
+repository, to make sure that if one mirror is down, another
+can be used.
+
+Here are some of the attributes of repository mirroring in {product-title}:
+
+* Image pulls are resilient to registry downtimes
+* Disconnected clusters can request to pull
+images from critical locations (such as quay.io)
+and have registries behind a company firewall
+provide the requested images.
+* A particular order of registries is tried when
+an image pull request is made, with the permanent
+registry typically being the last one tried.
+
+Setting up repository mirroring can be done in the following ways:
+
+* At {product-title} installation time: By pulling container images needed
+by {product-title} and then bringing those images behind your company's
+firewall, you can install {product-title} into a datacenter that is
+disconnected from the Internet. See
+link:https://github.com/openshift/openshift-docs/blob/636ca5e931e8e3000349649d86f280a814de25c4/modules/installation-mirror-repository.adoc[Mirroring the {product-title} image repository]
+for details.
+
+* After {product-title} installation time: Even if you don't configure mirroring during {product-title} 
+installation, you can do so later using the ImageContentSourcePolicy object.
+
+The following procedure provides a post-installation mirror configuration, where you create an ImageContentSourcePolicy object that identifies:
+
+* The source of the container image repository you want to mirror
+* A separate entry for each mirror repository you want to offer the content
+requested from the source repository.
+
+The mirror information you enter is added to the `/etc/containers/registries.conf`
+file on every node in the {product-title} cluster.
+
+When a node makes a request for an image from the source repository, it
+tries each mirrored repository in turn until it finds the requested on content.
+If they all fail, the cluster tries the source repository. Upon success, the
+image is pulled to the node.
+
+.Prerequisites
+* Have administrative privilege to a working {product-title} cluster.
```

Comment:
Do you mean 
```
* Access to the cluster as a user with the `cluster-admin` role.
```

If so, I'd say that - we use that language a couple of other places in the collection.

Reply:
Please add installing the oc CLI as a prereq.

We've been saying

```
* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
```

Reply:
Done.